### ISSUE TITLE
The "GenerateScript" task failed unexpectedly

### ISSUE THREAD

**Issue by ILAgent** [bridgedotnet/Bridge#827]
I've created new Bridge.NET class library project and tried to build - OK ! After that I've added my existed source files and tried to build. The result:

```
1>C:\Users\ilage\OneDrive\Src\PointsGame\packages\Bridge.1.10.4\build\Bridge.targets(12,5): error MSB4018: The "GenerateScript" task failed unexpectedly.
1>C:\Users\ilage\OneDrive\Src\PointsGame\packages\Bridge.1.10.4\build\Bridge.targets(12,5): error MSB4018: System.NullReferenceException: Object reference not set to an instance of an object.
1>C:\Users\ilage\OneDrive\Src\PointsGame\packages\Bridge.1.10.4\build\Bridge.targets(12,5): error MSB4018:    at Bridge.Contract.EmitterException.get_FileName()
1>C:\Users\ilage\OneDrive\Src\PointsGame\packages\Bridge.1.10.4\build\Bridge.targets(12,5): error MSB4018:    at Bridge.Build.GenerateScript.Execute()
1>C:\Users\ilage\OneDrive\Src\PointsGame\packages\Bridge.1.10.4\build\Bridge.targets(12,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
1>C:\Users\ilage\OneDrive\Src\PointsGame\packages\Bridge.1.10.4\build\Bridge.targets(12,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()
```
Have I done something wrong ?

**Comment by vladsch** [comment]
Is It reproduciable on simple example? Can you post source code which reproduces the issue?

**Comment by geoffreymcgill** [comment]
Please provide a code sample demonstrating how to reproduce.

**Comment by ILAgent** [comment]
[Dots.Bridge.zip](https://github.com/bridgedotnet/Bridge/files/96517/Dots.Bridge.zip)

Look please. I've tried to build it on two different PC with the same result.

**Comment by vladsch** [comment]
As I see you use C# 6 syntax a lot. Unfortunately, Bridge v1 supports C# 5 only
C# 6 will be supported in v2
We will improve notification about it (to avoid such ugly exception)

**Comment by ILAgent** [comment]
I've removed C# 6.0 syntax but I still see the same error
[Dots.Bridge.cs5.zip](https://github.com/bridgedotnet/Bridge/files/99156/Dots.Bridge.cs5.zip)
Look please.

**Comment by vladsch** [comment]
Your project still uses C# 6 in Rectangle.cs
Also i found one bug if create multidimensional array of structs (it is related with this code ` Points = new Dot[w, h];` in Game constructor)
I will close this issue and create another new for the found bug (the fix will be included to upcoming 1.11 release)
https://github.com/bridgedotnet/Bridge/issues/835
